kalman.py

```diff
@@ -1,53 +1,35 @@
-"""
-A gross oversimplification of the the kalman filter
-For each timestep:
-current estimation = Gain * Measured + (1 - Gain) * previousestimation
-
-The only unknown is the gain, this is continually revised using new
-info and something from the previous state Kalman filter finds the
-most optimum averaging factor for each consequent state. Also somehow
-remembers a little bit about the past states
-a very good dummy intro:
```

Comment:
could you add this link back in, I found it useful for understanding the code

Reply:
I have uploaded the 2 .py files. I didn't get you which link do u want?

Reply:
i added the link. Do check.

Reply:
Thankew so much by pushing it up.